docs/stage/download-raw-data.md

```diff
@@ -33,3 +36,19 @@ Run the SQL:
     DELETE FROM download_raw_errors;
     UPDATE publisher_feed SET raw_next_url=NULL;
     
+
+## To only download a limited set of raw data
+
+You may want to do this to avoid processing too much data. 
+Or to debug a problem that occurs with a specific publisher's data, you may want to get data for that publisher only.
+
+### Stop the process early
+
+The process can be forcibly stopped at any point and the database will not be in a bad state. 
```

Comment:
Will someone who's developing on this know how to forcibly stop?

Reply:
Yes. There's a argument we could add some notes to the Docker page but that's not for here.